docs/yaml.rst

```diff
@@ -4,8 +4,8 @@ Mapping YAML File
 =================
 
 The YAML mapping tells the BUFR component fields to read from the BUFR file, and how
```

Comment:
I'd rather use "which" - "The YAML mapping tells **which** BUFR component fields to read from the BUFR file, "

Reply:
Thanks! Done.